map/src/menu/actions/TrackActions.jsx

```diff
@@ -71,34 +70,34 @@ const TrackActions = forwardRef(({ track, setDisplayTrack, setZoomToTrack, setOp
         );
     };
 
-    const ShowOnMapAction = () => {
-        return (
-            <MenuItem
-                id="se-show-on-map-action"
-                className={styles.action}
-                onClick={() => {
-                    setZoomToTrack(true);
-                    setOpenActions(false);
-                }}
-            >
-                <ListItemIcon className={styles.iconAction}>
-                    <ShowOnMapIcon />
-                </ListItemIcon>
-                <ListItemText>
-                    <Typography variant="inherit" className={styles.actionName} noWrap>
-                        {t('shared_string_show_on_map')}
-                    </Typography>
-                </ListItemText>
-            </MenuItem>
-        );
-    };
+    // const ShowOnMapAction = () => {
+    //     return (
+    //         <MenuItem
+    //             id="se-show-on-map-action"
+    //             className={styles.action}
+    //             onClick={() => {
+    //                 setZoomToTrack(true);
+    //                 setOpenActions(false);
+    //             }}
+    //         >
+    //             <ListItemIcon className={styles.iconAction}>
+    //                 <ShowOnMapIcon />
+    //             </ListItemIcon>
+    //             <ListItemText>
+    //                 <Typography variant="inherit" className={styles.actionName} noWrap>
+    //                     {t('shared_string_show_on_map')}
+    //                 </Typography>
+    //             </ListItemText>
+    //         </MenuItem>
+    //     );
+    // };
```

Comment:
Show On Map action could co-exist together with click (the same approach is already implemented in Android).